src/stack.py

```diff
@@ -7,15 +7,17 @@ class Stack(object):
 
     def __init__(self, inbound_data=None):
         """Create new Stack composing from LinkedList."""
-        self.inbound_data = None
         self._linked_list = LinkedList(inbound_data)
+        self.head = self._linked_list.head
 
     def push(self, val):
         """Return a new node on the Stack using LinkedList.push."""
         self._linked_list.push(val)
+        self.head = self._linked_list.head
 
     def pop(self):
         """Return pop method for LinkedList on Stack."""
```

Comment:
This isn't returning a method. This is returning the value at the top of the stack. There's no need to make reference to the linked list that you're using to compose the stack. Just talk about the stack as if it is its own object.